### PR TITLE
Added componette info flash

### PR DESCRIPTION
**Pull request by f3l1x**
Look's like this.

da6wmgh

@chemix

### CODE REVIEW

**Comment by chemix** [comment]
I'm fine with it. :+1: 

Who can merge & release it? @milo @Vrtak-CZ ?

H.

File: app/templates/@layout.latte [hunk 1]
```diff
@@ -63,6 +63,7 @@
 		{block main}
 		<div class="main" lang="en">
 			<div class="content has-sidebar">
+				<div class="flash error">See our new modern addons portal <a href="http://bit.ly/cmtte">componette.com</a>. Completely automatic.</div>
```
Comment (fprochazka) [inline app/templates/@layout.latte]: I don't like that you redirect people from https site to https over HTTP redirect tool.

Comment (chemix) [inline app/templates/@layout.latte]: ok ... we can use something like 

'https://componette.com?from=addons.nette.org'  ?

Comment (fprochazka) [inline app/templates/@layout.latte]: I think that something like `https://componette.com/?utm_source=referrer&utm_medium=link&utm_campaign=addons.nette.org` is perfectly sufficient :) But either way is better.

Comment (Majkl578) [inline app/templates/@layout.latte]: https://bit.ly/cmtte works too, but :-1: for using external service for simple redirect.

Comment (f3l1x) [inline app/templates/@layout.latte]: We could change it to https. I agree.

But, the fact that you guys dont like it (bitly etc.), it's not a reason that we have to change it.

Comment (fprochazka) [inline app/templates/@layout.latte]: @f3l1x agreed, so I fixed it myself https://github.com/nette/web-addons.nette.org/commit/d4068434ee78a475c59bfac01caf5fcdc95cbff5

Comment (fprochazka) [inline app/templates/@layout.latte]: My point was not about the tool itself, but that you redirect of HTTP which is completely unnecesary and is in fact ilogical, when all three websites are on https.

**Pull request merged**